src/ome_zarr_models/cli.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import sys
+
+from ome_zarr_models import __version__, open_ome_zarr
+
+
+def main() -> None:
+    """Main entry point for the ome-zarr-models CLI."""
+    parser = argparse.ArgumentParser(
+        prog="ome-zarr-models",
+        description="OME-Zarr Models CLI",
+    )
+    parser.add_argument(
+        "--version",
+        action="version",
+        version=f"ome-zarr-models version {__version__}",
+    )
+    subparsers = parser.add_subparsers(
+        dest="command", help="Available commands", metavar="COMMAND"
+    )
+
+    # validate sub-command
+    validate_cmd = subparsers.add_parser("validate", help="Validate an OME-Zarr")
+    validate_cmd.add_argument("path", type=str, help="Path to zarr to validate")
+
+    # info sub-command
+    info_cmd = subparsers.add_parser("info", help="Get information about an OME-Zarr")
+    info_cmd.add_argument(
+        "path", type=str, help="Path to zarr to get information about"
+    )
+
+    args = parser.parse_args()
+
+    # Execute the appropriate command
+    match args.command:
+        case "validate":
+            validate(args.path)
+        case "info":
+            info(args.path)
+        case None:
+            parser.print_help()
+            sys.exit(1)
+
+
+def validate(path: str) -> None:
```

Comment:
It would be nice to add the option to valdate against a specific version and group-type. Something like:

```python
def validate(
    path: str,
    version: Literal["0.4", "0.5"] | None = None,
    group_type: Literal["image", "hcs", ...] | None = None
) -> None:
```

What do you think? Happy to punt that to a follow up PR, since it might need some more internal logic adding.